<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLoadSurfaceOverlay</class>
 <widget class="QDialog" name="DialogLoadSurfaceOverlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select volume file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditFile">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxRegistration">
     <property name="text">
      <string>Apply registration</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditRegistration">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRegistration">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLoadSurfaceOverlay</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLoadSurfaceOverlay</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRegistration</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditRegistration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRegistration</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonRegistration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadSurfaceOverlay</receiver>
   <slot>OnButtonOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRegistration</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadSurfaceOverlay</receiver>
   <slot>OnButtonRegistration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
  <slot>OnButtonOpen()</slot>
  <slot>OnButtonRegistration()</slot>
 </slots>
</ui>
